### PR TITLE
Mz modal edits

### PR DESCRIPTION
**Pull request by MarZmu**

### CODE REVIEW

File: client/src/components/QModal.jsx [hunk 1]
```diff
@@ -29,9 +29,9 @@ function QModal({ pID }) {
     //close window on submit
     alert(`${aModal ? 'Answer' : 'Question'} Submitted!`);
 
-    axios.get(`/a/questions/${pID}?count=${100}`)
-      .then((res) => res.data)
-      .catch((err) => { console.log(err); });
+    axios.get(`a/questions/${pID}?count=${100}`)
```
Comment (Dalphus) [inline client/src/components/QModal.jsx:32]: I believe the forward slash is required in front of this url. If it isn't, ignore this comment

File: client/src/components/QModal.jsx [hunk 2]
```diff
@@ -43,52 +43,54 @@ function QModal({ pID }) {
...
-        <br />
-        <br />
-        <label className='mod-label' htmlFor='nick'>
-          NickName *
           <br />
-          <input className='mod-input' name='nick' id='nick' placeholder='Ex: jackson11!' maxLength='60' required />
+          <label className='mod-label' htmlFor='newQA'>
+            { aModal ? 'Your Answer *' : 'Your Question *' }
+            <br />
+            <textarea className='q-text-area' rows='2' col='500' id='newQA' type='' maxLength='1000' name='newQA' placeholder={aModal ? 'Give us your two cents.. ' : 'What do you want to know about the product? '} required />
```
Comment (Dalphus) [inline client/src/components/QModal.jsx:65]: put the attributes for this element on their own lines

File: client/src/components/Question.jsx [hunk 3]
```diff
@@ -41,7 +41,8 @@ export default function Question({ question }) {
     .filter((a) => a.body.indexOf(aFilter) !== -1)
     .sort((a, b) => (b.helpfulness - a.helpfulness));
 
-  const loadAs = answers[2] && <button onClick={expand} className='a-load' type='button'>{exp ? 'COLLAPSE ANSWERS' : 'LOAD MORE ANSWERS'}</button>;
+  const moreAs = answers[2] !== undefined;
+  const loadAs = moreAs && <button onClick={expand} className='a-load' type='button'>{exp ? 'COLLAPSE ANSWERS' : 'LOAD MORE ANSWERS'}</button>;
```
Comment (Dalphus) [inline client/src/components/Question.jsx:45]: while cool, this syntax hurts readability. use an if statement or the ternary operator to achieve this functionality

**Review by Dalphus — CHANGES_REQUESTED** [review]
good stuff.
2 very small changes requested

File: client/src/components/QModal.jsx (hunk 1 above)
Comment (MarZmu) [reply client/src/components/QModal.jsx:32]: request works! :)

**Review by Dalphus — APPROVED** [review]

**Pull request merged**